Dnn.AdminExperience/Library/Dnn.PersonaBar.UI/admin/personaBar/css/main.css

```diff
@@ -469,34 +469,27 @@ span.dnnFormError:after {
     background-color: var(--dnn-color-pb-menu-background-hover);
 }
 
-.personabar .personabarLogo a.update {
+.personabar .personabarLogo .update {
     text-decoration: none;
     text-transform: uppercase;
     font-size: 10px;
     font-family: pb_semibold;
-    display: none;
     position: absolute;
     bottom: 8px;
     width: 50px;
+    border: none;
     background-color: var(--dnn-color-tertiary-light, #21a3da);
     border-radius: var(--dnn-controls-radius, 3px);
     color: var(--dnn-color-tertiary-contrast, #fff);
-    display: block;
+    display: none;

```

Comment:
Is this intended ?

Reply:
yes, it's hidden via CSS until Knockout kicks in an applies its `visible` binding.

Reply:
Ha, makes sense